OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -136,6 +136,7 @@ public class DefaultSpriteSequence : ISpriteSequence
 		public int Length { get; private set; }
 		public int Stride { get; private set; }
 		public int Facings { get; private set; }
+		public int InterpolatedFacings { get; private set; }
```

Comment:
I think we'll want this to be implemented on `SequenceProvider.ISpriteSequence`, so that places like `QuantizeFacingsFromSequence` can look up the value.
`QuantizeFacingsFromSequence` will then want to use `InterpolatedFacings` rather than `Facings` if its value is higher than `Facings`, otherwise the orientation of offsets will look off/coarse versus the sprite when actors turn (set a higher `InterpolatedFacings` on the RA cruiser `idle` sequence to see what I mean, the turrets will 'jump' because their position offset relative to the body is still calculated with 16 facings).

Reply:
This is a great suggestion, I have now implemented it and it has indeed smoothened the cruiser turret animation quite a bit, as shown below. There appears to be some glitchiness with the ship that isn't present with the aircraft, which I am guessing is because the original sprites are not at perfect angles relative to their facing, so it is probably not worth using InterpolatedFacings on the ships (or perhaps the number of facings should be reduced). But this change does now mean that turret facings interpolate with the unit itself, which is quite nice.

https://www.dropbox.com/s/9tu426nuynz1ugc/quantize_interp_facings.mp4?dl=0